Update -  [es_es.lang]

multiplefibers opened the pull request:
Added missing punctuations to advancements.
Many word corrections for the sake of consistency and coherence.
- may submit commits sporadically

multiplefibers committed:
Add Spanish (spain) localization

Derivated from the neutral spanish file, features advancements and required localization

multiplefibers committed:
es_es categories

multiplefibers committed:
es_es templates

multiplefibers committed:
es_es entries, first half

multiplefibers committed:
es_es entries, second half

multiplefibers committed:
Delete src/main/resources/assets/pyrotech/patchouli_books/book/es_es/entries directory

accident

multiplefibers committed:
es_es entries, first half

had deleted files because they all contained english macros, oops! now it's all right.

multiplefibers committed:
es_es entries, second half

multiplefibers committed:
Update book.json (es_es)

Adds Spanish (spain) macros

multiplefibers committed:
Update bloom_cog_machines.json

multiplefibers committed:
Small cohesion update (es_es)

Missing dots in advancements; touch-ups to the book.

multiplefibers committed:
Merge branch 'codetaylor:master' into master

multiplefibers committed:
Merge branch 'master' of https://github.com/eightyfoahh/pyrotech-1.12

multiplefibers committed:
Update es_es.lang

multiplefibers committed:
Update es_es.lang

Wood Chips (Serrín => Virutas)

codetaylor commented:
I intend to merge this and push a new build just as soon as I have time to setup my build environment again.

multiplefibers committed:
Book correction

Word changes (Serrín => Virutas) in two entries

multiplefibers committed:
Word corrections

Word changes for straw bales (Paca => Bala)

multiplefibers committed:
OPTIONAL: en_us change - Wood Chips

Wood Chips => Wood Chips Pile
Pile of Wood Chips => Wood Chips

+ All entries in the book already acknowledged the prior "Pile of Wood Chips" simply as "Wood Chips"

multiplefibers committed:
Word changes for Lard (es_es)

Manteca de cerdo => Manteca de animal

+ Lard was called "Pig Fat". You could harvest Pig Fat from bear carcasses.

multiplefibers committed:
Word changes for Marshmallow

Marshmallow => Malvavisco

multiplefibers committed:
Word changes for Lard

Manteca de animal => Grasa animal

multiplefibers committed:
Minor fixes: tooltips and WAILA related
